apps/explorer/src/components/top-validators-card/TopValidatorsCard.tsx

```diff
@@ -50,91 +78,122 @@ export const STATE_DEFAULT: ValidatorState = {
     },
 };
 
-export function TopValidatorsCardStatic() {
-    return <TopValidatorsCard state={mockState as ValidatorState} />;
+const textDecoder = new TextDecoder();
+
+export type ObjFields = {
+    type: string;
+    fields: any[keyof string];
+};
+
+export type SystemParams = {
+    type: '0x2::sui_system::SystemParameters';
+    fields: {
+        max_validator_candidate_count: number;
+        min_validator_stake: bigint;
+    };
+};
+
+export type ValidatorState = {
+    delegation_reward: number;
+    epoch: number;
+    id: { id: string; version: number };
+    parameters: SystemParams;
+    storage_fund: number;
+    treasury_cap: ObjFields;
+    validators: {
+        type: '0x2::validator_set::ValidatorSet';
+        fields: {
+            delegation_stake: bigint;
+            active_validators: Validator[];
+            next_epoch_validators: Validator[];
+            pending_removals: string;
+            pending_validators: string;
+            quorum_stake_threshold: bigint;
+            total_validator_stake: bigint;
+        };
+    };
+};
+
+function StakeColumn(validator: {
+    stake: BigInt;
+    stakePercent: number;
+}): JSX.Element {
```

Comment:
I'd dump the explicit `JSX.Element` return type here, and let inference take over.